apps/resources/sbt.json

```diff
@@ -3,13 +3,8 @@
     "central"
   ],
   "dependencies": [
-    "org.scala-sbt:sbt-launch:latest.stable",
-    "io.get-coursier.sbt:sbt-runner:latest.stable"
+    "org.scala-sbt:sbt:latest.stable"
   ],
-  "properties": {
-    "jline.shutdownhook": "false",
-    "jna.nosys": "true"
-  },
-  "javaOptions": ["-Xms512m", "-Xss2m"],
-  "jvmOptionFile": ".jvmopts"
```

Comment:
Same as above. We should keep those. These should be ignored by `cs install` anyway, when it's using the prebuilt launcher.